apps/src/templates/projects/projectsRedux.js

```diff
@@ -92,6 +92,18 @@ function personalProjectsList(state = initialPersonalProjectsList, action) {
         ...state,
         projects: action.personalProjectsList,
       };
+    case PUBLISH_SUCCESS:
+      var channelOfInterest = action.lastPublishedProjectData.channel;
+
+      var publishedProjectIndex = state.projects.findIndex(project => project.channel === channelOfInterest);
+
+      var updatedProjects = state.projects;
```

Comment:
I think this turned out really cleanly! 

On line 100, I think you're just creating a new reference to `state.projects`, and not actually creating a copy. That means that you're modifying and returning the same state, which you can't do in redux because `state` is meant to be read only.
Here's a thread about that: https://github.com/tur-nr/polymer-redux/issues/70

Apparently you can clone an array by doing a `.slice` operation: https://stackoverflow.com/questions/7486085/copying-array-by-value-in-javascript
So I think this line should be `var updatedProjects = state.projects.slice();`

Reply:
or `updatedProjects = [...state.projects]` if you want to be all es6y

Reply:
Even better!